nw-002approach.txt

```diff
@@ -0,0 +1,5 @@
+2. Baselines: established price by the company for full year, that can be used to compare current and future purchases, and evaluate the amount of spend and savings generated.
```

Comment:
This looks great! Let's go for it. Let's see if the data is normally distributed.

Reply:
Will do. Thanks!